### ISSUE TITLE
Dimmer SR-ZG9101SAC-HP didn’t work

### ISSUE THREAD

**Issue by Lowcostairpost** [easyhomesrc/homey_easylighting#3]
Hello

New dimmer with new chip SR-ZG9101SAC-HP didn’t work well. Old version SR-ZG9101SAC-HP with old chip that I tested worked good. When I tried to connect dimmer to Homey via your App the device connected as simple switch with On/Off function. Please fix this problem. 
Thank you

Best Regards,
Anton

**Comment by maginawin** [comment]
Hi @Lowcostairpost,

We have released the release version of EasyLighting App, here is the URL: https://homey.app/a/com.easyhome.easylighting/

Can you provide the Manufacturer ID and Product ID of this device? The device information is located in Device -> Settings -> Advance.

Best regards,
Wendong Wang

**Comment by iFac3** [comment]
I tried the new version what Athom released 10 hours ago. I have SR-ZG9040A-S and SR-ZG9100A-S. None of them recognised by the app, adds as basic zigbee devices. Manual switching is not reported back to Homey. HomeyKit adds dimmer module as switch to HomeKit, however in Homey both appears as dimmable modules.

**Comment by maginawin** [comment]
Hi @iFac3 ,

Did you install EasyLighting App v1.0.1 first and then add devices? If not, remove the device and add the device again.

I will get the corresponding devices later, and then confirm and solve this problem.

**Comment by iFac3** [comment]
Hi!

I removed and added an unit as a test, but result is the same, came up as a basic zigbee device.
(I use v1.0.1)

Rgds,

Gergely.

**Comment by maginawin** [comment]
@iFac3 

Can you provide the Manufacturer ID and Product ID of the devices? The device information is located in Device -> Settings -> Advance.

The devices I tested may be slightly different from yours.

**Comment by iFac3** [comment]
This is the dimmer module:
![IMAGE 2021-02-23 09:32:38](https://user-images.githubusercontent.com/51527573/108818717-131d8280-75ba-11eb-970a-6efeb2e3aeee.jpg)
![IMG_7515](https://user-images.githubusercontent.com/51527573/108818734-19abfa00-75ba-11eb-8b53-fa2b14b0b031.jpg)


Switch:

![IMAGE 2021-02-23 09:34:52](https://user-images.githubusercontent.com/51527573/108818915-64c60d00-75ba-11eb-8246-bd3145b961d0.jpg)

![IMG_8BC16D7CA657-1](https://user-images.githubusercontent.com/51527573/108818783-2c263380-75ba-11eb-8c28-e8725481c1ee.jpeg)

**Comment by maginawin** [comment]
Hi @iFac3 ,

I updated the App to support your devices, you can test it first, and it will release after passing the review.
https://homey.app/a/com.easyhome.easylighting/test/

**Comment by iFac3** [comment]
I’ve just checked both. All fine. Both can be added as EasyLight devices, reporting back status of manual switching or dimming work also fine!


Thanks!!!

Rgds,

Gergely.

**Comment by iFac3** [comment]
One more little thing. SR-ZG9100A-S module comes up as "Light" type device, however it could be other so a Type selector would be needed under Advanced Settings.

**Comment by maginawin** [comment]
Hi @iFac3 ,

Thank you for your suggestion, I will try to implement this feature.


Best regards,
Wendong Wang

**Comment by maginawin** [comment]
Hi @iFac3 ,

I found that only unknown type of device can chose device type, do you known other devices that support type switching?

**Comment by iFac3** [comment]
Yeah, actually it can be anything, since switch module can be installed behind wall AC outlet.

Rgds,

Gergely.

**Comment by maginawin** [comment]
Hi @iFac3 ,

The type of device is fixed and cannot be changed in the code. Do you know which Homey App can change the device type? I can learn how to do it.

**Comment by iFac3** [comment]
Hi!

Aqara app has T1 modules for example, both can be changed.






Rgds,

Gergely.
